src/coreclr/pal/src/eventprovider/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(EVENT_MANIFEST ${VM_DIR}/ClrEtwAll.man)
 
-if(CLR_CMAKE_HOST_LINUX)
+# TODO: LTTNG is not supported on Android. We need a solution.
```

Comment:
In lieu of a TODO, please file a github issue and provide a link in the comment.